irods/message/__init__.py

```diff
@@ -153,6 +153,16 @@ def ET(xml_type = 0, server_version = None):
 def _recv_message_in_len(sock, size):
     size_left = size
     retbuf = None
+
+    host, port = sock.getpeername()
```

Comment:
Get socket properties (host, port, is_blocking, and timeout) for debug messages and exception message.